.gitignore

```diff
@@ -44,6 +44,7 @@ node_modules
 test-report.html
 metadata.json
 edition.json
+/pro
```

Comment:
Will it still updates the submodule ref when we update pro?

Reply:
Yep since the pro folder is under /src/bp/pro and this line points to the root of the repo /pro